src/pages/Cart/Cart.css

```diff
@@ -61,11 +65,44 @@
   margin-bottom: 4px;
   font-size: 0.875rem;
 }
+.cart-item .coupon-details {
+  color: #2057a8;
```

Comment:
Suggestion: use variables

Reply:
I have updated it now. Thanks for pointing it out.